docs/workflow/testing/coreclr/unix-test-instructions.md

```diff
@@ -74,16 +74,23 @@ Is used to disable the build, that way if building on Unix cycles are saved buil
 PAL tests
 ---------
 
-Build CoreCLR on the Unix machine.
+Build CoreCLR with PAL tests on the Unix machine:
+
+```sh
+cd runtime
+./src/coreclr/build-runtime.sh -skipgenerateversion -nopgooptimize \
+    -cmakeargs -DCLR_CMAKE_BUILD_TESTS=1
+```
 
 Run tests:
 
-```
-~/runtime$ src/coreclr/src/pal/tests/palsuite/runpaltests.sh ~/runtime/artifacts/obj/coreclr/Linux.x64.Debug
+```sh
+# cd runtime
+src/coreclr/src/pal/tests/palsuite/runpaltests.sh $(pwd)/artifacts/obj/coreclr/$(uname).x64.Debug
 ```
 
 Test results will go into:
 
-```
-/tmp/PalTestOutput/default/pal_tests.xml
+```sh
+less /tmp/PalTestOutput/default/pal_tests.xml
```

Comment:
This change looks strange. The text above says "Test results will go into" so I would expect the file name and not a a command to dump it. I would suggest either keeping it as it was or placing the file name in single backtick to the end of `Test results will go into `

Reply:
Ah right, I was just copying these commands as steps, and thinking what is more useful for user. Changed so filename is inlined with text.